docs/guide/setup-examples/remote_dev.md

```diff
@@ -0,0 +1,189 @@
+This steps should help working remotely with Sublime Text.
+They are meant to be incremental,
+just setting up SFTP will go a long way.
+
+# My workflow
+
+I have all my code on my laptop (a Mac),
+edit locally and automatically push the files to my server (a Linux machine).
+I never edit "tracked" files on the server directly.
+I sometimes modify untracked files
+or config files on the server using `rmate` (see below).
+
+# SSH config
+
+Create a `~/.ssh/config` file,
+which allows reusing existing connection.
+This mean that you will have to authenticate only once.
+
+```
+Host server
+  Hostname SERVER_ADRESS
+  User MY_USERNAME
+  IdentityFile YOUR_KEY eg: ~/.ssh/id_rsa
+  AddKeysToAgent yes
+  UseKeychain yes
+  ControlPath ~/.ssh/ssh-%r@%h:%p
+  ControlMaster auto
+  ServerAliveInterval 240
+  Port 22
+```
+
+Check that it works by running "ssh server" in a terminal.
+Then in a second one.
+The second terminal shouldn't ask for credentials.
+
+This will also allow Sublime to connect to your server
+without asking for credentials.
+
+# SFTP setup
+
+SFTP is a Sublime Text package that will allow you to keep in sync files
+on your laptop and on a server.
+
+Install https://packagecontrol.io/packages/SFTP
+Open your project in ST.
+From command palette: **SFTP: Setup Server**
+This will create a `sftp-config.json` at the root of your project.
+(You may want to add `sftp-config.json` to your [global gitignore][].
+I recommend having the same project folder names
+on your server and on your laptop
+and keeping all your projects in the same folder.
+This will make advanced steps easier.
+
+Update the following keys:
+
+```json
+    // sftp, ftp or ftps
+    "type": "sftp",
+
+    "save_before_upload": true,
+    "upload_on_save": true,
+
+    "host": "server",
+    "user": "USER",
```

Comment:
Do these need to be set again when they are in the ssh-config already?

Reply:
if you don't set it SFTP will show an error everytime you try to interact with the server.
But I agree with you than in theory the user is not needed.